Updating docs (#2)
canonical/prometheus-configurer-k8s-operator: README.md

## Description

The Prometheus Configurer Charmed Operator provides an HTTP-based API for managing
[Prometheus](https://prometheus.io) alerting rules.

> [Juju](https://juju.is/) charm in this repository has been designed to supplement

Reviewer: ```suggestion
This charm has been designed to supplement the
```
